docs/superpowers/specs/2026-08-06-catalog-clean-architecture-design.md

```
@@ -0,0 +1,391 @@
+| `lib/clustering.ts` | `domain/model/cluster.ts`（或 domain/clustering） |
+| `lib/geo.ts` | `domain/geo/haversine.ts` |
+| `api/route.ts` 纯编排部分 | `application/plan-itinerary.ts` |
+| `api/route.ts` SQL | `adapters/outbound/neon/points-repo.ts` |
+| `api/route.ts` oRPC 入口 | `adapters/inbound/http/handlers/plan-itinerary.ts` |
+| `route` contract / path | **`itinerary`** |
+| `PilgrimagePoint` / `work_id` | **`Point` / `bangumi_id`** |
+| `route_snapshots` / 列 `work_id` | **`itinerary_snapshots` / `bangumi_id`** |
+| `api/search.ts` 等 | 同上模式拆 application + outbound |
+| `router.ts` / `index.ts` | `adapters/inbound/http/` |
+| `db/*` | `adapters/outbound/neon/` |
+| `ingest|enrich|publish` | `processes/*` 或 application 子包 |
```

markdownlint-cli2 (0.23.2)

[warning] Table column count
Expected: 2; Actual: 4; Too many cells, extra data will be missing

(MD056, table-column-count)